lib/spoom/cli/commands/coverage.rb

```diff
@@ -17,7 +17,9 @@ class Coverage < Thor
         def snapshot
           in_sorbet_project!
 
-          snapshot = Spoom::Coverage.snapshot(path: exec_path)
+          path = exec_path
+          snapshot = Spoom::Coverage.snapshot(path: path)
+          snapshot.sorbet_version = Spoom::Sorbet.srb_version(path: path)
```

Comment:
Super nitpicky, but I think I would prefer it more if the API was just:
```suggestion
          snapshot = Spoom::Coverage.snapshot(path: path)
```
and the snapshot process knew how to get the srb version by default.

Reply:
We can always read the version from the `Gemfile.lock` and get this done directly in the snapshot 👍